common/infrastructure/src/tracing.rs

```diff
@@ -123,15 +123,12 @@ fn init_otlp(name: &str) {
             "service.name",
             name.to_string(),
         )]))
-        .with_batch_exporter(exporter, opentelemetry_sdk::runtime::Tokio)
+        .with_batch_exporter(exporter, opentelemetry_sdk::runtime::TokioCurrentThread)
         .with_sampler(opentelemetry_sdk::trace::Sampler::ParentBased(Box::new(
             sampler(),
         )))
         .build();
 
-    println!("Using Jaeger tracing.");
```

Comment:
I'd love to keep those. As in the case here, we would not get logs on the console and in the pod logs, we would never know "why". With those, we do.

Reply:
np I deleted like normal printlns I'll add it back :+1: